herbert_core/classes/MPIFramework/@iMessagesFramework/iMessagesFramework.m

```diff
@@ -45,6 +45,8 @@
             if nargin>0
                 obj.job_id = varargin{1};
             else
+                % use process ID to seed the RNG so processes don't conflict
+                rng(feature('getpid'));
```

Comment:
this obj_id generation should be extracted to a util function, the same non-trivial code is now in two places

Reply:
we shouldn't reseed the random number generator before each call. seeds should be set once on each MATLAB process. If you're just wanting to generate a unique id for this job, concatenating PiD + datetime would give you something easily convertible to a string

Reply:
Yeah, I'm planning on finding the right place for this tomorrow. This push was a test to see if this fixes things